app/_locales/en/messages.json

```diff
@@ -1627,6 +1636,9 @@
   "welcome": {
     "message": "Welcome to MetaMask"
   },
+  "whatsthis": {
```

Comment:
Nit: This doesn't seem to follow our naming convention; it should be `whatsThis`

Reply:
addressed in 9bfe721f8